src/sentry/tasks/beacon.py

```diff
@@ -121,6 +122,7 @@ def send_beacon():
     # which is the same as False
     anonymous = options.get("beacon.anonymous") is not False
     event_categories_count = get_category_event_count_24h()
+    byte_to_gigabyte = 1024**-3
```

Comment:
I think you are mistaken
one byte / 1024^3 = one gigabyte
https://www.checkyourmath.com/convert/digital_storage/bytes_gigabytes.php

Reply:
pedantically this is a gibibyte not a gigabyte

Reply:
@hubertdeng123 I agree that 1024 bytes^3 is indeed a gigabyte(-ish, per anthony). My point is that taking it to the power of -3 here and multiplying below, rather than to the power of 3 and dividing, is more confusing.

Also, my 64-bit floating point is hazy, but I wonder if we wouldn't get precision errors for a significand that small (I think not, but still). We know for sure that 1GB can be contained in a 64-bit float, so seems easier to just use that and do simple division later?

Reply:
> pedantically this is a gibibyte not a gigabyte

TIL!

Hmmm, I can divide by a power of 3 instead if that's more clear. 

Not sure if I understand the second paragraph entirely @azaslavsky, but if the suggestion is to use GB instead of GiB, as far as I know in the context of physical memory everything I don't think we should need to worry about floating point numbers because memory should always be divisible by 1024 bytes.

Reply:
> Hmmm, I can divide by a power of 3 instead if that's more clear.

Seems more clear to me, but won't block on it.